packages/web/pages/user/my-account/technologies.js

```diff
@@ -145,4 +174,71 @@ export const NoTechnologies = styled.span`
 	font-size: 2rem;
 `;
 
+const statusModifiers = {
```

Comment:
This page should cover all technology statuses to avoid breaking styles:

![image](https://user-images.githubusercontent.com/13091635/102815387-0b1f4780-43ab-11eb-9980-1476422ec705.png)

Reply:
Agree. Do we have the style for the others statuses? Couldn't find it.

Reply:
@luizeboli I think we don't have those currently. @JoaoPauloSMoura could you give us a little help here, please?